### PR TITLE
PP-166 fix digital invoice line item wrong background color

### PR DESCRIPTION
**Pull request by igor-gini**

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Components/DigitalLineItemTableViewCell.swift [hunk 1]
```diff
@@ -21,6 +21,7 @@ class DigitalLineItemTableViewCell: UITableViewCell {
     @IBOutlet weak var editButton: UIButton!
     @IBOutlet weak var priceLabel: UILabel!
     @IBOutlet weak var separatorView: UIView!
+    @IBOutlet weak var backroundContainerView: UIView!
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Components/DigitalLineItemTableViewCell.swift:24]:
```suggestion
    @IBOutlet weak var backgroundContainerView: UIView!
```

**Review by zladzeyka — APPROVED** [review]
Thank you @igor-gini 🥳
Looks nice, just a small typo in var name.

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Thank you @igor-gini 🥳
Looks nice, just a small typo in var name.

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Components/DigitalLineItemTableViewCell.swift (hunk 1 above)
Comment (igor-gini) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Components/DigitalLineItemTableViewCell.swift]: Adjuster

**Review by zladzeyka — APPROVED** [review]
Thank you! 🙏 @igor-gini 
Please prepare a build for QA

**Pull request merged**